zap/src/main/java/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -852,7 +851,8 @@ public void saveParam(Object obj) throws Exception {
         options.getViewParam().setResponsePanelPosition(selectedItem.getPosition().name());
         options.getViewParam().setBrkPanelViewOption(brkPanelViewSelect.getSelectedIndex());
         if (brkPanelViewSelect.getSelectedIndex() == BreakLocation.TOOL_BAR_ONLY.getValue()) {
-            options.getParamSet(BreakpointsParam.class).setShowIgnoreFilesButtons(false);
+            options.getParamSet(org.zaproxy.zap.extension.brk.BreakpointsParam.class)
```

Comment:
This will lead to an exception as the `BreakpointsParam` is no longer being added.

Reply:
shall this be removed?

Reply:
Yes, this will have to be replaced (i.e. https://github.com/zaproxy/zaproxy/pull/8363#discussion_r1525917280).

Reply:
Bump.

Reply:
will it still lead to exception?

Reply:
I think this is a bug in github it isn't showing the outdated tag on it. you can check it from the changes once please @thc202

Reply:
okk thank you

Reply:
is it correct now?